### ISSUE TITLE
multicore_fifo_wready() docs might be backwards?

### ISSUE THREAD

**Issue by davidgiven** [raspberrypi/pico-sdk#235]
Filing as a bug rather than just sending a pull request because this might be intentional, but the documentation for `multicore_fifo_wready()` says:

```
Check the FIFO to see if the write FIFO is full.
Returns true if the FIFO is full, false otherwise
```

AFAICT it returns true if the FIFO is _not full_ (i.e. has at least one slot ready to write), which is what the name suggests. If this is correct, the documentation needs to be corrected; if not, the name probably wants changing!

**Comment by lurch** [comment]
Thanks, the [RP2040 datasheet](https://datasheets.raspberrypi.org/rp2040/rp2040-datasheet.pdf) says:
```
1    RDY     Value is 1 if this core’s TX FIFO is not full (i.e. if FIFO_WR is ready for more data)   RO    0x1
```
so it looks like this was a simple typo. Fixed by #239

**Comment by Wren6991** [comment]
Thanks, fixed
